app/models/job.rb

```diff
@@ -156,7 +156,7 @@ def execution
   end
 
   def status!(status)
-    success = update_column(:status, status)
+    success = update_attribute(:status, status)
```

Comment:
hmm now this can fail when deploy or job is invalid and result in jobs that are un-stoppable ... so would prefer the update_column + touch

Reply:
Doesn't `update_attribute` also bypass validations?